<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>591</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>33</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>131</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="prefix">
     <string>0x</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="displayIntegerBase">
     <number>16</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>131</height>
     </size>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>131</height>
     </size>
    </property>
    <property name="text">
     <string>Send </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRead">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>131</height>
     </size>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>131</height>
     </size>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Hex</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
